fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java

```diff
@@ -1603,6 +1624,20 @@ private boolean loadTxnCommitImpl(TLoadTxnCommitRequest request) throws UserExce
                         request.getTbl(), request.getUserIp(), PrivPredicate.LOAD);
             }
         }
+        if (request.groupCommit) {
+            String clientAddr = getClientAddrAsString();
+            SystemInfoService systemInfoService = Env.getCurrentSystemInfo();
+            long backendId = -1;
+            for (Long id : systemInfoService.getAllBackendIds(false)) {
+                Backend backend = systemInfoService.getBackend(id);
+                if (clientAddr.equals(backend.getHost())) {
+                    backendId = id;
+                    break;
+                }
+            }
```

Comment:
It's not a good idea to run a loop here, if there are 100 bes, then a high frequent load may leads to a problem.